idioms/clone-to-closure.md

```diff
@@ -0,0 +1,33 @@
+# Pass clone to closure
+
+## Description
+
+By default, closures capture their environment by borrowing. Or you can use `move`-closure
+to move environment. However, often you want to give copy of some data to closure.
+Use variable rebinding in separate scope for that.
+
+
+## Example
+
+```rust
+let mut num = Rc::new(5);
+let closure = {
+	let num = num.clone();
```

Comment:
This code should use four-space indents, not tabs, to stay consistent with other articles.